systemtests/tests/strippath/testrunner-03-verify

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+
+TestName=03-verify
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+# Create verify job
+cat <<END_OF_DATA > "$config_directory_dir_additional_test_config/strippath-verify-job.conf"
+Job {
+  Name = "verify-initcatalog-bareos-fd"
+  JobDefs = "DefaultJob"
+  Client = "bareos-fd"
+  FileSet = "SelfTestStripPath"
+  Type = Verify
+  Level = InitCatalog
+}
+Job {
+  Name = "verify-catalog-bareos-fd"
+  JobDefs = "DefaultJob"
+  Client = "bareos-fd"
+  FileSet = "SelfTestStripPath"
+  Type = Verify
+  Level = Catalog
+}
+Job {
+  Name = "verify-disktocatalog-bareos-fd"
+  JobDefs = "DefaultJob"
+  Client = "bareos-fd"
+  FileSet = "SelfTestStripPath"
+  Type = Verify
+  Level = DiskToCatalog
+  Verify Job = "$JobName"
+}
+Job {
+  Name = "verify-volumetocatalog-bareos-fd"
+  JobDefs = "DefaultJob"
+  Client = "bareos-fd"
+  FileSet = "SelfTestStripPath"
+  Type = Verify
+  Level = VolumeToCatalog
+}
+END_OF_DATA
```

Comment:
Same here: why does this happen at runtime ?

Reply:
Jup, that worked. Thanks!